homeassistant/components/iss/sensor.py

```diff
@@ -8,20 +8,23 @@
 import requests
 from requests.exceptions import HTTPError
 
-from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.components.sensor import SensorEntity
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import ATTR_LATITUDE, ATTR_LONGITUDE, CONF_SHOW_ON_MAP
+from homeassistant.const import (
+    ATTR_LATITUDE,
+    ATTR_LONGITUDE,
+    CONF_SHOW_ON_MAP,
+    STATE_UNKNOWN,
+)
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.util import Throttle
 
 _LOGGER = logging.getLogger(__name__)
 
-ATTR_ISS_NEXT_RISE = "next_rise"
 ATTR_ISS_NUMBER_PEOPLE_SPACE = "number_of_people_in_space"
```

Comment:
Remove?

Reply:
Seems that #65178 have been revived 😆 The next rise information is also gone unfortunately. 

Would it be better to use a device tracker for the location, and then split the rest on to sensors?

Reply:
Use number of people in space as the state, and lat and long as attributes ...

Reply:
@ktpx this is what I have changed it to?

Reply:
@ktpx @avee87 I see what you mean now... 😆